tools/local-environment/tests/agentsDieOnDoubleUpdate.ts

```diff
@@ -0,0 +1,228 @@
+import { LocalAgent } from "../src/agent";
+import { ethers } from "ethers";
+import { LocalNetwork, Nomad, Key, utils, Network } from "../src";
+
+import { Waiter } from "../src/utils";
+
+async function startAndTestAgent(
```

Comment:
nit: can we rename this something more related? Like `wait_agent_crash_loop` or `wait_agent_failure`?

Reply:
+1